horovod/run/run.py

```diff
@@ -333,15 +342,24 @@ def parse_args():
                              "HOROVOD_START_TIMEOUT can also be used to "
                              "specify the initialization timeout.")
 
-    parser.add_argument('--verbose', action="store_true",
+    parser.add_argument('--verbose', action="store",
                         dest="verbose",
-                        help="If this flag is set, extra messages will "
-                             "printed.")
+                        default=0, nargs='?', type=int,
+                        help="Can use 1 or 2 to set the verbosity level of "
+                             "debugging messages.")
 
     parser.add_argument('command', nargs=argparse.REMAINDER,
                         help="Command to be executed.")
 
-    return parser.parse_args()
+    parsed_args = parser.parse_args()
+
+    if parsed_args.verbose is None:
+        # This happens if the user does "horovodrun --verbose" without
+        # specifying any value to verbose. For the sake of consistency, we set
+        # the verbosity here to the default value of 2.
```

Comment:
Your comment mismatches the code.  I'd vote for keeping the code and fixing the comment.